src/style.css

```diff
@@ -1,26 +1,55 @@
+* {
+  font-family: "Poppins", sans-serif;
+}
+
+.body {
+  border: 2px solid;
+  margin-top: 100px;
+  margin-left: 220px;
+  margin-right: 220px;
+}
+
+h1 {
+  color: rgba(24, 91, 155, 0.87);
+}
+
```

Comment:
- [ ] For the style declarations you made in this file to reflect in the app, you need to import this stylesheet in the src/index.js file. Kindly import this style sheet and your style declarations will reflect in the application. 👍 😎 

### Your Application

![image](https://user-images.githubusercontent.com/77106962/233423802-f19d3cbc-aa72-45fc-86b7-6e0503c8d60a.png)